### ISSUE TITLE
Публикация сторонней, проприетарной библиотеки в свойбодном доступе

### ISSUE THREAD

**Issue by AleximusOn** [bocharnikov1969/itiscm#2]
Вы создали публичный репозиторий, в исходниках которого используется проприетарная внешняя копонента от серебренной пули:
https://github.com/bocharnikov1969/itiscm/blob/develop/src/CommonTemplates/V8RMQClient/Ext/Template.bin
 
Надеюсь это согласовано с @silverbulleters

**Comment by EvilBeaver** [comment]
Нет, это не согласовано. Вы допустили утечку компоненты в открытый доступ и нарушили лиц. договор. Срочно примите меры к удалению компоненты из общего доступа!

**Comment by EvilBeaver** [comment]
@AleximusOn спасибо!
